### ISSUE TITLE
Testing: #898 Spinboxes Create Assets

### ISSUE THREAD

**Issue by fdoving** [RavenProject/Ravencoin#984]
Post your test results as comments to this issue.

Please include release version or commit-hash used when testing.

**Comment by lorancecall** [comment]
27dec57

When reissue asset, cannot change the 'Unit'
Not sure this is originally designed this way.

**Comment by lorancecall** [comment]
27dec57
![image](https://user-images.githubusercontent.com/84116812/120088595-3c214800-c0c0-11eb-8a7a-26320985d8fa.png)

Invalid characters error during Qualifier Asset creation. 
- I only put capital alphabet, but error occured.

**Comment by lorancecall** [comment]
27dec57

On 'Overview' screen, right click of main and sub asset does not allow to create new asset type.

**Comment by fdoving** [comment]
Units can only be increased after creation. 8 is max.
This should probably be shown somewhere.

**Comment by lsji07** [comment]
Using Windows 10 and Raven Core version v4.7.0.0-b5010492c (64-bit)

Spin boxes work correctly in my testing. Most asset spinboxes are limited to 21 billion as necessary and unique assets are 1 as expected.

Agree that additional cautionary comment/information is required close to the units spinbox to clarify. something like. 

`Caution: Reissuing an asset token can only increase the units and not reduce them in future. Use the minimum units your usecase requires currently.` 

I also had issues with the alert message printing out of box. 
![image](https://user-images.githubusercontent.com/39772657/140646961-a90fdfbf-89c6-4169-aa81-bce887f00085.png)

#1045 was also fixed for me on testing.

**Comment by m4r1m0** [comment]
Tested and works correctly on Ubuntu 20.04 using raven-4.7.0test3-b5010492c (64 bit)

Tested spin boxes on Main Asset, Restricted Asset, Sub Asset, Qualifier Asset, and Sub Qualifier Asset. Unique Assets and Message Channel Assets have a quantity restricted to 1.

Was able to successfully reissue assets and change the units, but not the quantity (as expected/intended). 

The warning messages did clip on the GUI and I had to expand the window to read it. This should be fixed. When creating the qualifier I initially forgot the "#" and could not read the warning reminding me to include it.
